atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -193,6 +193,7 @@ const CreateCrashRecord = ({ client }) => {
             created_by: userEmail,
             cris_schema_version: "2023",
             is_temp_record: true,
+            private_dr_fl: false,
```

Comment:
default to  `private_dr_fl = false` on temp record creation, meaning the crash will show up in our socrata totals and in the crashes list page, etc

Reply:
perfect. glad you mentioned Socrata—I think we should the `is_temp_record` column to public datasets. i'll add it to the todo list :)